src/elements/Image/Image.js

```diff
@@ -73,6 +71,7 @@ function Image(props) {
     className,
   )
   const rest = getUnhandledProps(Image, props)
+  const [imgTagProps, rootProps] = partitionHTMLInputProps(rest, { htmlProps: imageProps })
```

Comment:
This seems strange to overload an input prop parser for an image.  Should this become a generic parser?

Reply:
Agree, I will update PR and we will go with `partitionProps`.